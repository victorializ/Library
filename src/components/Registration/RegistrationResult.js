import React from 'react';
import { Link } from 'react-router-dom';
import {
  Button,
  Header,
} from 'semantic-ui-react';

import { constants } from "../../i18n";
import { colors } from "../../assets/semantic-colors";

const RegistrationResult = () => 
  <>
    <Header textAlign="center" className='registered'>
      {constants.registered}
    </Header>
    <Button className='registered__login'
      as={Link} to='/login' 
      color={colors.primary}>
      {constants.login}
    </Button>
  </>

export { RegistrationResult };