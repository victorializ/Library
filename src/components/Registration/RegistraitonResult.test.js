import React from 'react';
import { shallow } from 'enzyme';

import { RegistrationResult } from './RegistrationResult';

describe('RegistrationResult component', () => {
    it('renders correctly', () => {
        expect(shallow(<RegistrationResult />)).toMatchSnapshot();
    }); 
});