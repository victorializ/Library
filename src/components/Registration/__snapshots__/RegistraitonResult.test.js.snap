// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationResult component renders correctly 1`] = `
<Fragment>
  <Header
    className="registered"
    textAlign="center"
  >
    Registered successfully
  </Header>
  <Button
    as={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "displayName": "Link",
        "propTypes": Object {
          "innerRef": [Function],
          "onClick": [Function],
          "replace": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      }
    }
    className="registered__login"
    color="olive"
    to="/login"
  >
    Login
  </Button>
</Fragment>
`;
